src/usr/local/pkg/miniupnpd.xml

```diff
@@ -192,12 +191,22 @@
 			<name>Advanced Settings</name>
 			<type>listtopic</type>
 		</field>
+		<field>
+			<fielddescr>Allow Third-Party Mapping</fielddescr>
+			<fieldname>allow_third_party_mapping</fieldname>
+			<description>
+				<![CDATA[
+				Allow adding port maps for non-requesting IP addresses
+				]]>
+			</description>
+			<type>checkbox</type>
+		</field>
 		<field>
 			<fielddescr>Download Speed</fielddescr>
 			<fieldname>download</fieldname>
 			<description>
 				<![CDATA[
-				Value to report when clients query the maximum link download speed (Kbit/s).
+				Value to report when clients query the maximum link download speed (kbit/s).
```

Comment:
The current text is fine since the configuration is generated by converting the GUI-provided Kb/s value into kb/s.